src/OrbitGl/App.cpp

```diff
@@ -2020,7 +2002,117 @@ orbit_base::Future<ErrorMessageOr<CanceledOr<void>>> OrbitApp::RetrieveModuleAnd
   return load_result;
 }
 
-orbit_base::Future<ErrorMessageOr<CanceledOr<std::filesystem::path>>> OrbitApp::RetrieveModule(
+Future<ErrorMessageOr<CanceledOr<std::filesystem::path>>> OrbitApp::RetrieveModuleViaDownload(
```

Comment:
`RetrieveModuleViaDownload` is only allowed to be called from the main thread, correct? Because it accesses `symbol_files_currently_downloading_` which is not thread safe. 

I haven't done this with other functions, but I think a `ORBIT_CHECK(std::this_thread::get_id() == main_thread_id_);` at the beginning of this functions makes sense.